packages/web3-providers-http/types/tsconfig.json

```diff
@@ -11,7 +11,7 @@
         "allowSyntheticDefaultImports": false,
         "baseUrl": ".",
         "paths": {
-            "web3-providers-http": ["."]
+            "web3-providers": ["."]
```

Comment:
This was completely misleading and if possible should we keep the``web3-providers-http`` path. We probably can have both paths to not break anything.

Reply:
I've resolved this and created a related issue (https://github.com/ethereum/web3.js/issues/3336)